src/main/scala/org/broadinstitute/dsde/workbench/gpalloc/monitor/ProjectCreationMonitor.scala

```diff
@@ -135,7 +147,7 @@ class ProjectCreationMonitor(projectName: String,
     status match {
       case CreatingProject => EnableServices
       case EnablingServices => CompleteSetup
-      case _ => throw new WorkbenchException("what the")
+      case _ => throw new WorkbenchException(s"ProjectCreationMonitor for $projectName called getNextStatusMessage with surprising status $status")
```

Comment:
i think this may potentially be triggered when resuming projects from the db. #17